saskatoon/harvest/views.py

```diff
@@ -277,3 +276,69 @@ def harvest_yield_create(request):
             messages.success(request, _("New Fruit Recipient successfully added!"))
 
         return HttpResponseRedirect(request.META.get('HTTP_REFERER'))
+
+
+@login_required
+def harvest_adopt(request, id):
+    """
+    Adds harvest pickleader and changes status to Adopted
+    Used in a button at harvest-detail/status template
+    """
+    harvest = get_object_or_404(Harvest, id=id)
+    current_user = request.user
+
+    if harvest.pick_leader == None:
```

Comment:
I believe PEP8 discourages ` == None`,  you could use `if harvest.pick_leader is None:` instead (or even `if not harvest.pick_leader:` in this case).

Reply:
Thanks for the headsup.
I think `if not harvest.pick_leader:` is also [not a good practice](https://peps.python.org/pep-0008/#programming-recommendations)

Reply:
Thanks for the headsup.
I think if not harvest.pick_leader: is also [not a good practice](https://peps.python.org/pep-0008/#programming-recommendations)